src/components/DappBrowser/Homepage.tsx

```diff
@@ -144,14 +145,30 @@ const Card = React.memo(function Card({
   goToUrl,
   site,
   showMenuButton,
+  index,
 }: {
   goToUrl: (url: string) => void;
   showMenuButton?: boolean;
   site: Omit<Site, 'timestamp'>;
+  index?: number;
 }) {
   const { isDarkMode } = useColorMode();
+  const browserHistoryStore = useBrowserHistoryStore();
   const { dapps } = useDapps();
 
+  const dappClickedBefore = useMemo(() => browserHistoryStore.hasVisited(site.url), [browserHistoryStore, site.url]);
+
+  const handlePress = useCallback(() => {
+    {
+      index &&
+        analyticsV2.track(analyticsV2.event.browserTrendingDappClicked, {
+          hasClickedBefore: dappClickedBefore,
```

Comment:
why track this? i don't think this info is significant/worth tracking but maybe i'm wrong

Reply:
yeah for this one, it was an ask on the ticket: https://linear.app/rainbow/issue/APP-1496/track-engagement-with-trending-dapps